widget.subscribe.php

```diff
@@ -413,6 +415,38 @@ function add_subscriber() {
 				}
 			}
 
+            $recaptcha_token = sanitize_text_field( $_POST['captcha_token'] );
```

Comment:
Fix whitespace to match the tab/spacing in front of the code

Reply:
The spacing isn't off for me in my local copy of the code...weird

Reply:
My guess is it's an issue with tabs? Converting to spaces fix it everywhere.